addons/web/static/src/core/browser/cookie_service.js

```diff
@@ -43,6 +43,7 @@ function makeCookieService() {
         get current() {
             return cookie;
         },
+        parseCookieString,
```

Comment:
no, if you need the function, simply export it. it's pure, it should not be part of the service